File: missingnumber.py
# to find missing number in the sequence
def missingnumber(nums): # make function
    n=len(nums)+1 #total numbers including missing number
    sum_of_n = n*(n+1)//2 # formula to calculate sum of first n numbers
    sum_of_nums=sum(nums)
    missingnumber = sum_of_n - sum_of_nums
    return missingnumber  # return the result
print(missingnumber([1,2,3,4,6,7,8])) # print it

File: missingnumber.java
public class missingnumber {
   public static void main(String[] args){
    int total;
    int[] number=new int[]{1,2,3,5,6,7};
    total=7;
    int expected_sum=total*((total+1)/2);
    int num_sum =0;
    for (int i: number){
        num_sum +=i;

    }
    System.out.println(expected_sum-num_sum);
   }
}
